doc/user_guide/statisticalAnalysisExample.tex

```diff
@@ -157,4 +157,3 @@ \section{Statistical Analysis through RAVEN}
 \end{tabular}
 \end{table}
 
-\input{statisticsToDataObject.tex}
```

Comment:
Did you mean to remove this?

Reply:
Okay...I saw your answer above...

Reply:
Yes, that was valid before the dataobject rework. It is not now.